src/content/docs/en/concepts/islands.mdx

```diff
@@ -20,39 +39,44 @@ The term "Astro Island" refers to an interactive UI component on an otherwise st
+    <Fragment slot="source">Source: [Islands Architecture: Jason Miller](https://jasonformat.com/islands-architecture/)</Fragment>
 </IslandsDiagram>
 
-In Astro, you can use any supported UI framework (React, Svelte, Vue, etc.) to render islands in the browser. You can mix and match different frameworks on the same page, or just pick your favorite.
+An island always runs in isolation from other islands on the page, and multiple islands can exist on a page. Islands can still share state and communicate with each other, even though they run in different component contexts.
 
-The technique that this architectural pattern builds on is known as **partial** or **selective hydration.** Astro leverages this technique behind the scenes, powering your islands automatically. 
+This flexibility allows Astro to support multiple supported UI frameworks like [React](https://react.dev/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), and [SolidJS](https://www.solidjs.com/). Because they are independent, you can even mix several frameworks on each page.
 
-## How Do Islands Work in Astro?
+:::tip
+Although most developers will stick to just one UI framework, Astro supports multiple frameworks in the same project. This allows you to:
+
+- Choose the framework that is best for each component.
+- Learn a new framework without needing to start a new project.
+- Collaborate with others even when working in different frameworks.
+- Incrementally convert an existing site to another framework with no down time.
+:::
+
+## Creating an island
+
+By default, Astro will automatically render every UI component to just HTML & CSS, **stripping out all client-side JavaScript automatically.** 
 
-**Astro generates every website with zero client-side JavaScript, by default.** Use a frontend UI component built with [React](https://react.dev/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/), or [Lit](https://lit.dev/) and Astro will automatically render it to HTML ahead of time and then strip out all of the JavaScript. This keeps every site fast by default by removing all unused JavaScript from the page.
 
 ```astro title="src/pages/index.astro"
----
-// Example: Use a static React component on the page, without JavaScript.
-import MyReactComponent from '../components/MyReactComponent.jsx';
----
-<!-- 100% HTML, Zero JavaScript loaded on the page! -->
 <MyReactComponent />
 ```
 
-But sometimes, client-side JavaScript is required for creating interactive UI. Instead of forcing your entire page to become a single-page JavaScript application (SPA), Astro asks you to create an island.
+This may sound strict, but this behavior is what keeps Astro websites fast-by-default and protects developers from accidentally sending unnecessary or unwanted JavaScript that might slow down their website.
```

Comment:
In the Why Astro page, and the old Islands page, we use "Fast by default" (no hyphen), I would change this for consistency.
```suggestion
This may sound strict, but this behavior is what keeps Astro websites fast by default and protects developers from accidentally sending unnecessary or unwanted JavaScript that might slow down their website.
```